app/src/store/slices/validator-slice.ts

```diff
@@ -10,7 +9,7 @@ const { defaultPagination } = loadConfig();
 
 export interface ValidatorState {
   status: Loading;
-  validators: ValidatorWeight[];
+  validators: ApiData.Validators['validators']['validators'];
```

Comment:
Could this be written as `validators: ValidatorsInfo[]`?

Reply:
😅 yup, and it most certainly should. I don't think accessing nested types like this is all that great. Thanks!

Reply:
Awesome!